bin/i18n/sync-out.rb

```diff
@@ -165,7 +165,7 @@ def restore_redacted_files
       elsif original_path.starts_with? "i18n/locales/original/course_content"
         # Course content should be merged with existing content, so existing
         # data doesn't get lost
-        restored_data = RedactRestoreUtils.restore_file(original_path, translated_path, ['blockly'])
+        restored_data = RedactRestoreUtils.restore_file(original_path, translated_path, ['blockly', 'startHtml'])
```

Comment:
When using multiple plugins, I have found in the past that the order in wich they are used matters for some edge cases.
Have you checked that the content is restored as expected?

Reply:
yep! startHtml should go 2nd because blockly redacts `<xml>` tags differently than startHtml would